pkg/machines/getOSList.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python3
+
+import gi
+gi.require_version('Libosinfo', '1.0')
+from gi.repository import Libosinfo
+import sys
+
+
+loader = Libosinfo.Loader()
+loader.process_default_path()
+db = loader.get_db()
+
+oses = db.get_os_list()
+for i in range(oses.get_length()):
+    os = oses.get_nth(i)
```

Comment:
This looks very unpythonic. You can't just iterate more naturally over `for os in oses:`?

Reply:
That's a gobject introspection limitation, unfortunately.

Reply:
Okay, I've worked this around in the same way done by virt-install.

Reply:
If it's a GI limitation, it's fine to keep. Still better than the large overhead of defining the helper class, IMHO.

Reply:
Right, I'll move it back to the way it was.

For the future, in case Cockout starts relying on Libosinfo for resources/devices, I do believe that the helper class may become handy. However, for now, let me just get back to the original proposal. :-)